corehq/apps/export/models/new.py

```diff
@@ -975,6 +975,7 @@ class InferredSchema(Document):
     group_schemas = SchemaListProperty(InferredExportGroupSchema)
     case_type = StringProperty(required=True)
     version = IntegerProperty(default=1)
+    last_app_versions = DictProperty()
```

Comment:
ah makes sense. Might be nice to put that in a comment in the code. I imagine this being confusing to someone looking at it in the future.